tools/PTE/pte-execRequest.js

```diff
@@ -1727,6 +1691,11 @@ function eventRegisterBlock() {
                             tx_stats[tx_evtTimeout]++;
                         }
                         evtRcv = evtRcv + 1;
+                        // BlockMetadataIndex_TRANSACTIONS_FILTER = 2
+                        if (block.metadata.metadata[2][i] !== 0) {
```

Comment:
What does "0" represent? Is this equivalent to "VALID"? Maybe we should define a constant for it. Reference the value from the fabric repo (API).

Reply:
agree